docs/visualization/plotting-overview.ipynb

```diff
@@ -202,28 +246,24 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "d.coords['xlabels'] = sc.Variable(dims=['x'], values=np.arange(M) + 15.)\n",
-    "sc.plot(d['data_2d'], labels={'x': 'xlabels'})"
+    "da.transpose().plot()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<div class=\"alert alert-info\">\n",
-    "\n",
-    "*Note*\n",
-    "\n",
-    "The `labels` dict uses *dimension labels* as keys, i.e., the `'x'` above refers to the `x` dimension and *not* the x-axis of the matplotlib figure.\n",
-    "\n",
-    "</div>"
+    "When the Matplotlib \"widget\" backend is activate it is also possible to transpose the axes via a button in the toolbar."
```

Comment:
This is not yet true with `plopp`, and I am not sure I want to support this.

Reply:
This is the scipp documentation, which currently is about scipp's plotting, not plopp.